### ISSUE TITLE
Accessibility: Lens-back anchor has no label

### ISSUE THREAD

**Issue by sedlatschek** [laravel/nova-issues#3394]
- Laravel Version: 8.41.0
- Nova Version: 3.25.0
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version Win10
- Browser type and version: Chrome 88 / Edge 90

### Description:

The `lens-back` anchor has no label. [JAWS 2021](https://www.freedomscientific.com/products/software/jaws/) reads it as "Arrow left, heading level 1 link". It would be nice to have something more descriptive such as "Back to :resource table" as title or some `sr-only` element.

![lens-back](https://user-images.githubusercontent.com/58665536/118531490-2e75d500-b746-11eb-9260-3b39ee151dc8.png)

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.
